### DISCUSSION TITLE
Is it possible to replay TTS audio?

### DISCUSSION THREAD

**Discussion by TFWol** [LostRuins/koboldcpp#1136] [Q&A] [answered]
I may be overlooking something, but I can't find anything. 
Is there a simple replay button somewhere to have the Text to Speech audio repeated?

**Comment by LostRuins** [comment, marked as answer]
Unfortunately not, currently you'll have to regenerate the text to make it play again

**Reply by TFWol** [reply]
Drat. Oh well.
Thanks for the reply so I can stop searching.
